### ISSUE TITLE
[OFFTOPIC] Any info about ProctorExam?

### ISSUE THREAD

**Issue by lathianamc** [nxvvvv/safe-exam-browser-bypass#22]
I'm very sorry for being offtopic and I understand this repo is for SEB, but, since I have nowhere to ask, I thought I might get a shot here.

My uni decided that they will proceed with ProctorExam for the distance exams from now on. I would really appreciate it if someone has any knowledge of ProctorExam (by Turnitin) and if it would work via the VM @nxvvvv posted.

Thank you in advance and again, sorry for hijacking the repo with an off-topic issue.

UPDATE:
@nxvvvv I see that you have added a way for people to sponsor you, and since I was the second person to support that idea by another member here, I feel not only obliged but also delighted to support your effort. I've sent you a small token of appreciation, thanks for your work on SEB, unfortunately, my Uni won't be using SEB, but still, you've done a great job!

**Comment by nxvvvv** [comment]
Hey, thank you so much for your generous support, it means a lot. 

Regarding the ProctorExam, it would be nice if you could share the setup file since I don't have access to it and I didn't find any setup files online. I'll try my best to make a workaround for you but yea I'm not assuring it.

**Comment by lathianamc** [comment]
Hey, @nxvvvv! Thanks a lot for the offer but don't take it the wrong way, my donation wasn't for you to spend your time examining another software, it was purely because I wanted to support you even with a small donation. There isn't any setup file, it seems that it is just a chrome extension that monitors your camera and your screen.

I will close this topic to not disrupt the main topic of the repository since a conversation can continue even with a closed issue.

What I have found so far is that with my modified VM where I expose the host hardware to the guest (besides GPU obviously), I can pass all the checks of the ProctorExam in the demo system that my university has published. This means that with the VM that @nxvvvv has uploaded (which I have not tested but I saw that he takes the same approach as I did for avoiding VM detection), you most likely (at least at the time of writing this post), be able to bypass any restriction.

What I strongly suggest to you though is that from my understanding, ProctorExam may have some loose security because it monitors your screen and your camera with AI. Your screen and your camera recording will be stored for a short period. Any "strange" behaviour will be marked as a timestamp so either your lecturer or a third-party entity will investigate and decide if your exam was a product of cheating or not. Having said that, I encourage you to prepare for your exams as best as possible and use the techniques shared here only as an assistance in case you forget something. Personally, I'm having a hard time memorising 30 - 50 math formulas and besides that, I find it obscured to have to memorise such things, especially when I know how to use and the meaning of every one of the formulas, therefore, on a second monitor I want to have the formulas so I can see them and be able to focus more on solving the problem rather than examining my memory abilities. In addition to that, I don't like having others lurking on my computer where I have personal and work stuff.

I hope this will help others in the future. I'm still looking into it because I don't see if there is a mechanism that prevents you from running other apps, I guess since you share your screen which is recorded it kind of eliminates the need to have a mechanism for preventing you from running other apps, I guess. In any case, any new finds I'll be posting them here.
